### ISSUE TITLE
[Bug] Portrait mode assets are cropped in the asset share page

### ISSUE THREAD

**Issue by artdirk** [Adobe-Marketing-Cloud/asset-share-commons#84]
Portrait mode assets are cropped such that only their centers are shown within the asset share page.

**Card layout**
![assetsharecardlayout](https://user-images.githubusercontent.com/33265694/33863975-45f82b52-de9f-11e7-82f4-d6b9161a5c4e.jpg)

**List layout**
![assetsharelistlayout](https://user-images.githubusercontent.com/33265694/33863976-46144062-de9f-11e7-80b0-d047bbd99dec.jpg)

**Comment by davidjgonzalez** [comment]
Correct - do you have a better universal solution?

**Comment by artdirk** [comment]
I will try a few things out and suggest a solution if it handles both portrait and landscape images properly. Thanks for the quick response!

**Comment by davidjgonzalez** [comment]
Happy to incorporate - this was the hang that seemed to look best across sizes that we found. We didn’t like variable height cards since they tended to make uneven rows which are hard to scan.

Something we could do as well is add a “thumbnail style” to the results components that supports a few different approaches so users can choose if it’s more important to them.

**Comment by davidjgonzalez** [comment]
Hang = best (not edit of comment on phone ;))

**Comment by artdirk** [comment]
I looked into this, and just changing the object-position property from "center" to "top" improves things quite a bit.

**Card layout**
Updated the object-position property from the .ui.cards > .card > .image > img, .ui.card > .image > img style.
![assetsharecardlayoutfix](https://user-images.githubusercontent.com/33265694/33915917-00919348-df5b-11e7-9d5d-50e79c2e0284.jpg)


**List layout**
Updated the object-position property from the .ui.table tbody tr td.image:not(.ui) img style.
![assetsharelistlayoutfix](https://user-images.githubusercontent.com/33265694/33915932-153e2842-df5b-11e7-8b38-0ce39a98e695.jpg)

I'm not sure if this change will have implications anywhere else, but it looks pretty good in my local AEM 6.3 SP1 development environment. :-)

**Comment by davidjgonzalez** [comment]
Wonder if this should be configurable...   i guess this really depends on the composition of the image. position top looks funny for things that have less content at the top of the image (of course)


![image](https://user-images.githubusercontent.com/1451868/34058770-55cf135c-e1aa-11e7-9347-c0e2413726d5.png)

![image](https://user-images.githubusercontent.com/1451868/34058814-6f959d9c-e1aa-11e7-87d0-e2cb9eb4b610.png)

**Comment by artdirk** [comment]
I totally agree about images with less content at the top, and this begs an interesting question my manager had...would it make sense to allocate more of the card real estate to the image and less to the title, data and actions? If this were a viable option it might mitigate the issue if the object-position property were changed from "center" to "top"...thoughts?

**Comment by godanny86** [comment]
yes that could be a good enhancement to offer a few different thumbnail size options. Not sure if you have seen this @artdirk: https://adobe-marketing-cloud.github.io/asset-share-commons/pages/development/guide/ -> Extend Search Results Renderer. Its fairly easy to create a custom Card / List renderer to add/modify the metadata displayed but could also be used to change the size of the card thumbnail.

**Comment by davidjgonzalez** [comment]
To allow authors to configure the size.. im wondering how that might look since since we don't have a way to "Author" different views; you just pick the renderer.

![image](https://user-images.githubusercontent.com/1451868/34088233-fe71aa02-e375-11e7-866c-5fe7489eb033.png)

It seems like the most reasonable option would be to add 2 more options to the above that are

Thumbnail size, Thumbnail position (top, center, bottom, cover, etc.)

Size gets interesting since we can't do pixels since the card and list views thumbs wont be the same height; so we could do a % off whatever is the "default heights" (50%, 100%, 200%, 500% etc) or we could create some arbitrary list of sizes (Small, Medium, Large) and each is set to some predefined height (Not sure what granularity of sizes would make everyone happy - if that's possible)
